zulipterminal/ui_tools/boxes.py

```diff
@@ -1361,6 +1361,26 @@ def main_view(self) -> List[Any]:
         ]
         return [part for part, condition in parts if condition]
 
+    def update_msg_content_header(self) -> bool:
```

Comment:
As you note in the commit (also in the next one), this is not a class method. Class methods are not bound to objects.